src/components/CountriesList.module.scss

```diff
@@ -13,4 +13,5 @@
 .list {
   display: flex;
   flex-wrap: wrap;
+  justify-content: space-around;
```

Comment:
It will create another issue with a useless gap between the main container and cards.